integration/server.ts

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as wrangler from 'wrangler';
+import * as fs from 'fs';
+import * as path from 'path';
+
+function deleteFolderSync(dirPath: string): void {
+  if (fs.existsSync(dirPath)) {
+    fs.readdirSync(dirPath).forEach((file: string) => {
+      const curPath = path.join(dirPath, file);
+      if (fs.lstatSync(curPath).isDirectory()) {
+        // Recursively delete contents
+        deleteFolderSync(curPath);
+      } else {
+        // Delete file
+        fs.unlinkSync(curPath);
+      }
+    });
+    fs.rmdirSync(dirPath);
+  }
+}
+
+export default async function start() {
```

Comment:
this can only be run if you use node (coz wrangler does not support bun as of today), and you can only use TS/Jest with node if you bring in Jest and TS as dependancies, but I personally do not think a tidier integration test is worth the overheads, but its your call. I this code is much nicer than the one it is replacing so maybe we go with this and wait for wranger to fix compatibility with bun.